includes/admin/class-edd-notices.php

```diff
@@ -660,6 +660,19 @@ private function add_user_action_notices( $notice = '' ) {
 						'class'   => 'error'
 					) );
 					break;
+				case 'address-removed' :
+					$this->add_notice( array(
+						'id'      => 'edd-customer-address-removed',
+						'message' => __( 'Customer address removed', 'easy-digital-downloads')
```

Comment:
```suggestion
						'message' => __( 'Customer address deleted.', 'easy-digital-downloads')
```

`deleted` matches the "Delete" action link.